github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
[Suggested Test Cases:] It would be helpful to add some test cases that cover edge cases or unexpected inputs for the `openai` package.